defect-tracker-server/src/main/java/com/defect/tracker/util/ValidationConstance.java

```diff
@@ -2,7 +2,21 @@
 
 public class ValidationConstance {
 
-	public static final String PROJECT_EXISTS = "Project  Already Exists";
+	public static final String PROJECT_EXISTS = "project  already Exists";
+
+	public static final String EMPLOYEE_EMAIL_EXISTS = "employee email  already Exists";
+
+	public static final String EMPLOYEE_Id_EXISTS = "employee id  already Exists";
```

Comment:
please change  first letter of  all words to upper case

Reply:
ok